src/adapters/email.go

```diff
@@ -0,0 +1,34 @@
+package adapters
+
+import (
+	"fmt"
+	"net/smtp"
+)
+
+type EmailAdapter struct {
```

Comment:
In this task, we use the MVC pattern. But I expect that at the level of business logic, we will use something like services. But why are you writing Adapter here? Could you explain, please? As far as I know, an adapter is a programming pattern (https://refactoring.guru/design-patterns/adapter), but I've never seen this suffix written at the service level. Wouldn't it be better to use EmailService?